ios-i-book-lover/Extensions/Date+Extension.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Date.swift
+//  BookLover
+//
+//  Created by nguyen.nam.khanh on 4/11/19.
+//  Copyright © 2019 nguyen.nam.khanh. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension Date {
+    func getHalfYearAgo() -> String {
```

Comment:
```suggestion
    func getHalfYearAgo() -> Date {
```
như này sẽ hợp lý hơn, khi nào cần string thì gọi hàm toString() bên dưới